SCF/SCF-Session8-ZooProblem/Cage.java

```diff
@@ -0,0 +1,71 @@
+import java.util.*;
+
+public class Cage {
+
+	private String animalCategory;
```

Comment:
I think there is mentioned in question that a cage is for specific type of animal and zone for a specific type. hence property should be of type animaltype.